helm-chart/kuberay-operator/README.md

```diff
@@ -18,10 +18,12 @@ version.BuildInfo{Version:"v3.6.2", GitCommit:"ee407bdf364942bcb8e8c665f82e15aa2
 
 To avoid duplicate CRD definitions in this repo, we reuse CRD config in `ray-operator`:
 ```console
-$ kubectl apply -k "github.com/ray-project/kuberay/ray-operator/config/crd?ref=v0.3.0"
+$ kubectl create -k "github.com/ray-project/kuberay/ray-operator/config/crd?ref=v0.3.0"
```

Comment:
minor: The PR should be `apply -> create`?

Reply:
oops, fixed